tasks/eva_3090/archive_to_lts.py

```diff
@@ -0,0 +1,98 @@
+import gzip
+import shutil
+import tarfile
+import os.path
+from argparse import ArgumentParser
+
+from ebi_eva_common_pyutils.logger import logging_config
+from retry import retry
+
+logging_config.add_stdout_handler()
+logger = logging_config.get_logger(__name__)
+
+
+def make_tarfile(output_filename, source_dir):
+    with tarfile.open(output_filename, "w") as tar:
+        tar.add(source_dir, arcname=os.path.basename(source_dir))
+
+
+def is_compressed(file_path):
+    compressed_ext = ['.gz', '.zip', '.bz', '.tbi', '.csi']
+    for ext in compressed_ext:
+        if file_path.endswith(ext):
+            return True
+    return False
+
+
+def compress(src_file_path, dest_file_path):
+    MEG = 2 ** 20
+    with open(src_file_path, 'rb') as f_in:
+        with gzip.open(dest_file_path, 'wb') as f_out:
+            shutil.copyfileobj(f_in, f_out, length=16 * MEG)
+
+
+def matches(name, patterns):
+    return any((pattern for pattern in patterns if pattern in name))
+
+
+@retry(tries=5, delay=3, backoff=2, logger=logger)
+def retriable_remove(path):
+    shutil.rmtree(path)
+
+
+def archive_directory(root_dir, scratch_dir, destination_dir, filter_patterns=None):
```

Comment:
```suggestion
def archive_directory(source_dir, scratch_dir, destination_dir, filter_patterns=None):
```

Reply:
Addressed that in (https://github.com/EBIvariation/eva-common-pyutils/pull/33)